packages/interactivity-router/src/index.js

```diff
@@ -128,23 +191,30 @@ export const { state, actions } = store( 'core/router', {
 		 * needed, and updates any interactive regions whose contents have
 		 * changed. It also creates a new entry in the browser session history.
 		 *
-		 * @param {string}  href                               The page href.
-		 * @param {Object}  [options]                          Options object.
-		 * @param {boolean} [options.force]                    If true, it forces re-fetching the URL.
-		 * @param {string}  [options.html]                     HTML string to be used instead of fetching the requested URL.
-		 * @param {boolean} [options.replace]                  If true, it replaces the current entry in the browser session history.
-		 * @param {number}  [options.timeout]                  Time until the navigation is aborted, in milliseconds. Default is 10000.
-		 * @param {boolean} [options.loadingAnimation]         Whether an animation should be shown while navigating. Default to `true`.
-		 * @param {boolean} [options.screenReaderAnnouncement] Whether a message for screen readers should be announced while navigating. Default to `true`.
+		 * @param {string|Object} eventOrUrl                         The page href or the event handler in case it is used directly in a directive.
```

Comment:
There is type-checking included later which defines the exact type:
```suggestion
		 * @param {string| Event} eventOrUrl                         The page href or the event handler in case it is used directly in a directive.
```

Reply:
Changed as part of [this commit](https://github.com/WordPress/gutenberg/pull/59707/commits/f7e5212961ad3909abc3b6665f4a1f06bc66a3f6).